internal/tui/filemanager/config.go

```diff
@@ -5,10 +5,21 @@ import (
 	"path/filepath"
 
 	"github.com/signadot/cli/internal/config"
-	"github.com/signadot/libconnect/common/trafficwatch/api"
 )
 
-type OnNewLineCallback func(metaRequest api.RequestMetadata)
+type MessageType string
+
+const (
+	MessageTypeData            MessageType = "data"
+	MessageTypeStatusNoStarted MessageType = "target_not_found"
+)
+
+type LineMessage struct {
+	MessageType MessageType
+	Data        any
```

Comment:
I think we should

1) have another field for the error, say `Error error`.
2) make Data a string

The tagged union pattern will work fine here.